### DISCUSSION TITLE
Error running `npx projen` after `eslint`

### DISCUSSION THREAD

**Discussion by automartin5000** [projen/projen#3186] [Q&A]
After running `projen eslint`, I'm all the sudden getting the following error when attempting to run `npx projen`:
```
Default task skipped. Trying legacy synthesis since --rc is specified
Synthesis failed: Unable to find a task named "default"
```

Not sure if the `eslint` execution is related or not.

**Comment by mrgrain** [comment]
What's your `.projen/tasks.json` file looking like? Does it really not have a `default` task and if so why?

**Reply by automartin5000** [reply]
I see this in `tasks.json`:
```Typescript
    "default": {
      "name": "default",
      "description": "Synthesize project files",
      "steps": [
        {
          "exec": "ts-node --project tsconfig.dev.json .projenrc.ts"
        }
      ]
    },
```

**Reply by automartin5000** [reply]
Aw man, I was not even paying attention (trying to get a commit out before lunch) and in the wrong directory. Sorry about that 🤦

**Comment by automartin5000** [comment]
> What's your `.projen/tasks.json` file looking like? Does it really not have a `default` task and if so why?
